.github/workflows/main.yml

```diff
@@ -400,21 +450,6 @@ jobs:
         os: ['windows-2019']
         vs-toolset: ['14.0', '14.2']
         python-version: [ '3.6', '3.9', '3.10' ]
-        # Must use windows-2016 image because it installs VS2017 (MSVC 14.1)
-        # Scons cannot find MSVC 14.1 when VS2019 is installed
```

Comment:
As far as I know this bug hasn't been resolved by SCons yet... All the 14.0 and 14.1 builds are failing and it finds MinGW (I assume) as the compiler: https://github.com/Cantera/cantera/runs/5337563240?check_suite_focus=true#step:6:57

Reply:
For VS 2022, it appears that only 14.3 and 14.0 works for SCons on my local machine, so the issue may have become more severe ...

Reply:
It seems to find the latest installed version among 14.1,14.2, and now 14.3. At least, that was my interpretation when I looked into this last.

Reply:
SCons does get past the initial steps, but for whatever reason `cl` is not found once the compilation starts. Looks like two issues are overlapping ... :man_shrugging: